packages/docs/docs/tutorials/bots/hl7-into-fhir.md

```diff
@@ -80,13 +80,16 @@ When we this implementation is complete we will:
 
     // Return Ack
     return input.buildAck().toString();
-}
+  }
+```
+
+Functionally, the code above will create a new patient with the `mrnNumber` provided, assuming that patient isn't aready in this system.
 
 ### Testing your Bot
 
 You'll need your bot id (see [bot list](https://app.medplum.com/Bot) and click) to execute the bot. Once you have found it, you can attempt to execute your Bot using an HTTP message by sending the following via curl.  Note the content type.
 
-```cURL
+```console
```

Comment:
Use "bash" rather than "console"

There is a specific list of languages supported, which affects syntax highlighting.  We should only use a supported language.  If the language we want isn't on the list, then we should leave off the language tag, and just 3 backticks.

https://github.com/github/linguist/blob/master/lib/linguist/languages.yml